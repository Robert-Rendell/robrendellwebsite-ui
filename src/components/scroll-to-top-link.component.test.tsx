import React from "react";
import { render } from "@testing-library/react";
import { ScrollToTopLink } from "./scroll-to-top-link.component";

describe("ScrollToTopLink component", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<ScrollToTopLink/>);
    expect(asFragment()).toMatchSnapshot();
  });
});