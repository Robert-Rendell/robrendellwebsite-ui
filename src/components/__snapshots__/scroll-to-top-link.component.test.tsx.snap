// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTopLink component should match snapshot 1`] = `
<DocumentFragment>
  <span>
    Back toÂ 
    <a
      class="scroll-to-top"
    >
      top of page
    </a>
    .
  </span>
</DocumentFragment>
`;
